buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt

```diff
@@ -10,7 +10,7 @@ object IntellijVersionHelper {
    * eapOfLatestSupportedMajor)
    * @returns Corresponding release numbers.
    */
-  fun resolveIntelliJVersions(versionKey: String?): List<String> {
+  fun resolveIntelliJVersions(versionKey: String?, intellijVersions: IntellijVersions): List<String> {
```

Comment:
Actually... can this be made an instance method of `IntellijVersions` now?

Reply:
Sure, why not